src/components/pages/lessons/LessonInfo.tsx

```diff
@@ -60,22 +60,24 @@ const TweetLink: FunctionComponent<{
   }
   className?: string
 }> = ({lesson, instructor, className = ''}) => {
+  const encodeTweetUrl = () => {
```

Comment:
made this so it is parseable by human, for some reason it was affecting my syntax highlighting in VS Code